sites/public/src/lib/helpers.tsx

```diff
@@ -307,3 +313,84 @@ export const getSiteHeader = (router: NextRouter) => {
     />
   )
 }
+
+export const getSiteFooter = () => {
+  return (
+    <SiteFooter>
+      <FooterSection sectionClassName="justify-around" small>
+        <div className="text-left">
+          <img
+            className="h-20 w-20 mr-3"
+            src="/images/bahfa-logo.png"
+            alt="Bay Area Housing Finance Authority Logo"
+          />
+          <img
+            className="h-24 w-96"
+            src="/images/mtc-abag-logo.svg"
+            alt="Association of Bay Area Governments - Metropolitan Transporation Commission Logo"
```

Comment:
@ifranch looks like we're missing a `t` in Transpor**t**ation

Reply:
Done, thanks for spotting that